client/src/App.js

```diff
@@ -14,15 +16,25 @@ export default class App extends Component {
 
   addToSavedList = movie => {
     const savedList = this.state.savedList;
-    savedList.push(movie);
-    this.setState({ savedList });
+    const findMovie = savedList.find(el => movie.id === el.id);
```

Comment:
thank you for refactoring this function :) It looks a million times better than the given.